Gruntfile.js

```diff
@@ -122,7 +125,102 @@ module.exports = function(grunt) {
     },
   });
 
-  grunt.registerTask('build', ['less','jekyll:build']);
+  grunt.registerTask('buildMeetupPosts', 'Fetches all events from Meetup.com and creates Jekyll posts where they dont exist', function(){
+    var done = this.async();
+    var allEventsQuery = "https://api.meetup.com/2/events?offset=0&format=json&limited_events=False&group_id=17778422&photo-host=secure&page=100&fields=&order=time&status=past%2Cupcoming&desc=false&sig_id=153356042&sig=66887b77c34d304571d20465b10229ce582b7e02";
+    var total = 0, complete = 0;
+
+    function postComplete(){
+      if(++complete === total){
+        done();
+      }
+    }
+
+    function formatDate(date){
+      var dateString = '';
+      dateString += date.getFullYear()+'-';
+      if(date.getMonth()+1<10){
+        dateString += '0';
+      }
+      dateString += (date.getMonth()+1);     
+      dateString += '-';
+      dateString += date.getDate();
+      return dateString;
+    }
+
+    function formatDateWithTime(date){
+      var dateString = '';
+      dateString += date.getFullYear()+'-';
+      if(date.getMonth()+1<10){
+        dateString += '0';
+      }
+      dateString += (date.getMonth()+1);     
+      dateString += '-';
+      dateString += date.getDate() + " ";
+      if(date.getUTCHours()<10){
+        dateString += "0";
+      }
+      dateString += date.getUTCHours() + ":";
+      if(date.getMinutes()<10){
+        dateString += "0";
+      }
+      dateString += date.getMinutes() + ":";
+      if(date.getSeconds()<10){
+        dateString += "0";
+      }
+      dateString += date.getSeconds();
+      return dateString;
+    }
+
+    function getFilename(post){
+      var date,title,filename;
+
+      date = new Date(post.time);
+      title = post.name.replace(/ /g, '-');
```

Comment:
[slug](https://github.com/dodo/node-slug) might be a cooler way to do this (and emoji support🎉)